### PR TITLE
Default priority for all tasks (task_default_priority setting)

### PR DESCRIPTION
**Pull request by madprogrammer**
## Description

There is a `task_queue_max_priority` configuration setting to specify default max priority for all queues, but there is no `task_default_priority` setting to allow specifying an implicit default priority for all tasks.

I wanted all my tasks to implicitly have a "middle" default priority of 5 (while having x-max-priority of 10), and to be able to selectively tune some specific tasks to have a different priority. So I've added such functionality in this pull request. It has been tested in production and seems to work as designed.

### CODE REVIEW

**Comment by madprogrammer** [comment]
@auvipy Test added

**Comment by madprogrammer** [comment]
@auvipy issue fixed, coverage should be ok too now

File: t/unit/tasks/test_tasks.py [hunk 1]
```diff
@@ -774,6 +778,97 @@ def yyy2():
...
+        @self.app.task(shared=False, priority=66)
+        def yyy4():
+            pass
+
+        @self.app.task(shared=False, bind=True, base=TaskWithPriority)
+        def yyy5(self):
+            pass
+
+        self.app.conf.task_default_priority = 42
+        old_send_task = self.app.send_task
```
Comment (georgepsarakis) [inline t/unit/tasks/test_tasks.py]: I don't think you need to save and restore the method object, `self.app` is reinstantiated on each test setup.

File: docs/userguide/configuration.rst [hunk 2]
```diff
@@ -1923,6 +1923,16 @@ Default: :const:`None`.
 
 See :ref:`routing-options-rabbitmq-priorities`.
 
+.. setting:: task_default_priority
+
+``task_default_priority``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+:brokers: RabbitMQ
```
Comment (georgepsarakis) [inline docs/userguide/configuration.rst]: Although this is a indeed a standard feature of RabbitMQ, the Redis transport also emulates priorities.

**Comment by georgepsarakis** [comment]
@madprogrammer several Travis CI failures have already been resolved in current master branch, could you please rebase?

File: t/unit/tasks/test_tasks.py (hunk 1 above)
Comment (madprogrammer) [reply t/unit/tasks/test_tasks.py]: The restoration part is there because I saw same code in other existing test, and copied it into my test

**Pull request merged**